doc-QuickStart_OpenStack_Provider_Guide/topics/Configuring_CloudForms.adoc

```diff
@@ -0,0 +1,62 @@
+[[configuring_a_database]]
+=== Configuring a Database for {product-title}
+
+Before using _{product-title}_, you will need to configure the database options for it. _{product-title}_ provides two options for database configuration:
+
+* Install an internal PostgreSQL database to the appliance
+* Configure the appliance to use an external PostgreSQL database
+
+[NOTE]
+=======
+See link:https://access.redhat.com/documentation/en/red-hat-cloudforms/version-4.0/deployment-planning-guide/#cpu_sizing_assistant_for_a_dedicated_vmdb_host[CPU Sizing Assistant for a Dedicated VMDB Host] in the Deployment Planning Guide for guidelines on CPU requirements.
+=======
+
+
+=== Configuring an Internal Database
+
+[IMPORTANT]
+======
+Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, Red Hat recommends adding the disk before starting the appliance. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
+======
+
+. Start the appliance and open a terminal from your virtualization or cloud provider.
+. After starting the appliance, log in with a user name of `root` and the default password of `smartvm`. This displays the Bash prompt for the `root` user.
+. Enter the `appliance_console` command. The _{product-title}_ Appliance summary screen displays.
```

Comment:
File removed as part of: https://github.com/ManageIQ/manageiq_docs/pull/184/commits/87068f77677f6a031edab31278c1f5272f15b5e0

Didn't need the file, I was planning to remove it in a subsequent commit.